projects/fsastorefrontlib/src/assets/translations/de/document.de.ts

```diff
@@ -0,0 +1,12 @@
+export const document = {
+  document: {
```

Comment:
We need a central place for document localization. Currently, it was distributed over multiple files